deploy/crds/pulumi.com_stacks.yaml

```diff
@@ -773,7 +773,8 @@ spec:
                 type: boolean
               workspaceReclaimPolicy:
                 description: |-
-                  WorkspaceReclaimPolicy specifies whether the workspace should be deleted or retained after the Stack is synced.
+                  WorkspaceReclaimPolicy specifies whether the workspace should be deleted or retained after the Stack is
+                  successfully synced, and the last update is in a succeeded state.
```

Comment:
I'm struggling with the phrase "successfully synced".  May I suggest "WorkspaceReclaimPolicy specifies whether the workspace should be deleted or retained after the Stack is successfully updated".